src/views/CampusSafety/components/Header/index.tsx

```diff
@@ -1,66 +1,90 @@
-                Gordon
-              </Box>{' '}
-              Campus Safety Resources
-            </Typography>
-            <Typography className={styles.subtitle}>
-              <i>"Helping students help themselves"</i>
-            </Typography>
+// Define the props for Header
+type HeaderProps = {
+  safetyPage?: boolean;
+  children?: React.ReactNode;
+};
+
+const Header: React.FC<HeaderProps> = ({ safetyPage = true, children }) => {
+  const location = useLocation();
+  const pathnames = location.pathname.split('/').filter((x) => x);
+
+  if (safetyPage) {
+    return (
+      <>
+        <Grid container alignItems="center" columnSpacing={4} className={styles.headerMain}>
+          <Grid item container xs={9} alignItems="center" columnSpacing={2}>
+            <Grid item xs={8}>
+              <Typography className={styles.title}>
+                <Box component="span" sx={{ color: 'secondary.main' }}>
+                  Gordon
+                </Box>{' '}
+                Campus Safety Resources
+              </Typography>
+              <Typography className={styles.subtitle}>
+                <i>"Helping students help themselves"</i>
+              </Typography>
+            </Grid>
           </Grid>
         </Grid>
-      </Grid>
-      <AppBar className={styles.stickyNav}>
-        <Breadcrumbs aria-label="breadcrumb">
-          {/* Home breadcrumb */}
-          <CampusSafetyBreadcrumb link={location !== '/campussafety' ? `/campussafety` : null}>
-            <Grid container alignItems="center">
-              <HomeIcon sx={{ mr: 0.5 }} fontSize="inherit" />
-              {'Campus Safety Home'}
-            </Grid>
-          </CampusSafetyBreadcrumb>
-        </Breadcrumbs>
-      </AppBar>
-    </>
-  ) : (
-    children && <Grid className={styles.mainHeader}>{children}</Grid>
-  );
+        <AppBar className={styles.stickyNav}>
+          <Breadcrumbs
+            aria-label="breadcrumb"
+            separator={<NavigateNextIcon fontSize="small" />}
+            className={styles.breadcrumbContainer}
+          >
+            {/* Home breadcrumb */}
+            <CampusSafetyBreadcrumb link={pathnames.length > 0 ? '/campussafety' : null}>
+              <Grid container alignItems="center">
+                <HomeIcon sx={{ mr: 0.5 }} fontSize="inherit" />
+                {'Campus Safety Home'}
+              </Grid>
+            </CampusSafetyBreadcrumb>
+
+            {/* Only render additional breadcrumbs beyond "Campus Safety Home" */}
+            {pathnames
+              .slice(1) // Start from the second item to avoid redundancy
+              .map((value, index) => {
+                const isLast = index === pathnames.length - 2;
+                const to = `/${pathnames.slice(0, index + 2).join('/')}`;
+
+                return (
+                  <CampusSafetyBreadcrumb key={to} link={!isLast ? to : null}>
+                    {value.replace(/-/g, ' ')}
```

Comment:
What is the purpose of the .replace here?  None of our routes have dashes in them, so why are we replacing dashes with spaces?

Reply:
I had that there because i was thinking of adding hyphens to the links that were showing on the frontend, just so it looks cleaner, so if we had lets say Lost-And-Found, it would show as Lost And Found on the breadcrumb rather than what it is currently lostandfound all together. But we can change that if we think its not neccessary but that was my initial thought for having that in there

Reply:
Gotcha, that's what I was wondering, and that makes sense to leave it in for now.  One thing we will have to deal with, and this was an issue I didn't catch in a previous PR, but urls should be all lowercase (by convention, and by what everything else on the web does), so the routes we wrote will need to be changed to be that way too, so you may have to write some fancier formatting code to capitalize them down the line (we don't need to fix that rn).